api/client/samples/crop_models/Sao_Paulo_brazil_sugar_content_modeling.ipynb

```diff
@@ -0,0 +1,3618 @@
+{
```

Comment:
<p>We import several packages later on in this notebook. Can we do all the imports at the start of the notebook, so that if it fails it fails early?</p><p> </p><p>There should also be a requirements.txt doc that lists all of the needed packages for this model.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/gro-intelligence/api-client/pull/76/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='1'/>

Reply:
<p>Done</p>